SUPPORT.md

```diff
@@ -28,9 +28,9 @@ people offer support.
 * Also check out the 
 [Slack Archive](http://kubernetes.slackarchive.io/) of past conversations.
 
-### Mailing Lists/Groups
+### Forum
 
-* [Kubernetes-users group](https://groups.google.com/forum/#!forum/kubernetes-users)
+* [Kubernetes Official Forum](https://discuss.kubernetes.io)
```

Comment:
:+1: 

@mrbobbytables can you also remove the slack archive note?

Reply:
Also, the comment in the end says:

```
<!---
Derived from https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-user-support.md
--> 
```

This can be removed since the link is 404.